TODO/VMware.json

```diff
@@ -1,6 +1,6 @@
 {
     "version": "14.1.2-8497320",
-    "description": "Industry standard for running multiple operating systems as virtual machines (VMs) on a single Linux or Windows PC.",
+    "description": "VMWare. Industry standard for running multiple operating systems as virtual machines (VMs) on a single Linux or Windows PC.",
```

Comment:
I would prefer we add a `name` property. Agreed?

Reply:
Not sure if would be usable for more people. I am doing this only for my custom search cmdlet for my end-users.